### ISSUE TITLE
Breaking update on version 1.5.0 - Issue with ibis_teradata/compiler.py StringIO usage

### ISSUE THREAD

**Issue by aymanfarhat** [GoogleCloudPlatform/professional-services-data-validator#335]
After updating from version 1.4.0 to 1.5.0, the tool is breaking on pretty much any command related to teradata querying.

Example command

```
data-validation -v validate column \
-sc TD_DVT \
-tc BQ_DVT \
-tbls TEST_DB.USA_NAMES_CORRECT=TEST_DB.USA_NAMES_CORRECT \
-bqrh rich-torus-323508.DVT_RESULTS.sample \
--avg=repeated
```

Output

```
/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/pandas/core/frame.py:1549: FutureWarning: Using short name for 'orient' is deprecated. Only the options: ('dict', list, 'series', 'split', 'records', 'index') will be used in a future version. Use one of the above to silence this warning.
  warnings.warn(
-- ** Source Query ** --
Traceback (most recent call last):
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/bin/data-validation", line 8, in <module>
    sys.exit(main())
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/__main__.py", line 379, in main
    validate(args)
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/__main__.py", line 358, in validate
    run(args)
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/__main__.py", line 339, in run
    run_validations(args, config_managers)
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/__main__.py", line 315, in run_validations
    run_validation(config_manager, verbose=args.verbose)
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/__main__.py", line 304, in run_validation
    validator.execute()
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/data_validation.py", line 94, in execute
    result_df = self._execute_validation(
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/data_validation.py", line 247, in _execute_validation
    source_query = validation_builder.get_source_query()
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/data_validation/validation_builder.py", line 260, in get_source_query
    print(query.compile())
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/expr/types.py", line 243, in compile
    return compile(
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/client.py", line 409, in compile
    return backend.compile(
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/client.py", line 256, in compile
    return query_ast.compile()
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/backends/base_sqlalchemy/compiler.py", line 55, in compile
    compiled_queries = [q.compile() for q in self.queries]
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/backends/base_sqlalchemy/compiler.py", line 55, in <listcomp>
    compiled_queries = [q.compile() for q in self.queries]
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/ibis/backends/base_sqlalchemy/compiler.py", line 1650, in compile
    select_frag = self.format_select_set()
  File "/Users/aymanf/projects/teradata-experiments/tdvt-1.5/lib/python3.8/site-packages/third_party/ibis/ibis_teradata/compiler.py", line 188, in format_select_set
    buf = StringIO()
NameError: name 'StringIO' is not defined
```

Looked like StringIO needs an import on `third_party/ibis/ibis_teradata/compiler.py`. Adding `from io import StringIO` into the file fixed the issue. 

As per the Python docs, [in Python 3 StringIO is no more included by default](https://docs.python.org/3.0/whatsnew/3.0.html#text-vs-data-instead-of-unicode-vs-8-bit). "The StringIO and cStringIO modules are gone. Instead, import the io module and use io.StringIO or io.BytesIO for text and data respectively." Might be a detail that was missed in this case.

Thanks!
